engine/render_particles.py

```diff
@@ -85,14 +87,19 @@ def main():
             continue
         else:
             print('rendering...')
-        Path(output_fn).touch()
+
         t = time.time()
 
-        renderer.set_camera_pos(3.24, 1.86, -4.57)
+        renderer.set_camera_pos(3.24, 2.86, -4.57)
```

Comment:
```suggestion
        renderer.set_camera_pos(3.24, 1.86, -4.57)
```

Please add an option for the user to specify the camera position.

Reply:
how about we move this feature to another PR

Reply:
Of course. We can keep this one as-is. Small changes are good.